docs/index.rst

```diff
@@ -8,6 +8,8 @@ Contents:
 .. toctree::
    :maxdepth: 3
 
-   jdbc_connector
-   configuration_options
+   source_connector
```

Comment:
Yeah, maybe why it looks a bit weird is that we get a level of hierarchy within the first page such that when you generate the docs you get a `+` symbol for that with a bunch of subsections, and then the configs are separate.